4/Step 1/Shipment.ts

```diff
@@ -0,0 +1,53 @@
+import {Shipper} from "../Step 2/Shipper";
+
+export interface ShipmentInfo {
+    ShipmentID: number;
+    Weight: number;
+    FromAddress: string;
+    FromZipCode: string;
+    ToAddress: string;
+    ToZipCode: string;
+}
+
+const PRICE_PER_OUNCE = 0.39;
```

Comment:
If this is only used in `Shipment` class you can instead use as private static class proprty.